lib/core/include/irods/guiProgressCallback.h

```diff
@@ -1,6 +1,9 @@
 #ifndef _GUI_PROGRESS_CALLBACK_H__
 #define _GUI_PROGRESS_CALLBACK_H__
 
+#include "irods/rodsDef.h"
+#include "irods/rodsType.h"
```

Comment:
Consider adding a trailing comment after each header indicating why it's needed.

For example:
```cpp
#include "irods/rodsDef.h"  // For MAX_NAME_LEN
#include "irods/rodsType.h" // For rodsLong_t
```

We don't do this for everything, but it can be helpful when the files are fairly small and it's not clear why a particular header is needed.